tests/integration/transactions/test_did_delete.py

```diff
@@ -0,0 +1,31 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+from tests.integration.it_utils import (
+    sign_and_reliable_submission_async,
+    test_async_and_sync,
+)
+from tests.integration.reusable_values import WALLET
+from xrpl.models.response import ResponseStatus
+from xrpl.models.transactions import DIDDelete, DIDSet
+
+_VALID_FIELD = "1234567890abcdefABCDEF"
+
+
+class TestDIDSet(IntegrationTestCase):
+    @test_async_and_sync(globals())
+    async def test_basic(self, client):
+        # Create DID to delete
+        setup_tx = DIDSet(
+            account=WALLET.address,
+            did_document=_VALID_FIELD,
+            uri=_VALID_FIELD,
+            data=_VALID_FIELD,
+        )
+        response = await sign_and_reliable_submission_async(setup_tx, WALLET, client)
+        self.assertEqual(response.status, ResponseStatus.SUCCESS)
+
```

Comment:
Can you add checks to see that the DID object was created / deleted like in the js version? (While that kind of is testing rippled, I'm worried about the `ResponseStatus` not necessarily 100% being the same as the transaction being applied - I'm not sure if some error codes count as a successful response, e.g. ter codes which could be applied in a future ledger)

Reply:
Done